fontforge/parsettf.c

```diff
@@ -2796,6 +2796,10 @@ return( 3 );
 	pt = buffer;
 	do {
 	    ch = getc(ttf);
+		if ((pt-buffer) > (sizeof(buffer) - 3)) {
```

Comment:
Thanks for reviewing! I've slightly modified the check in addition to adding in the comments (technically, it should only check for `sizeof(buffer) - 2`).